Submissions/Yolo86/Task_1/csvtojson.py

```diff
@@ -0,0 +1,25 @@
+import csv
+import json
+import os
+
+path1 = os.getcwd()
+for i in range(3):
+	path1 = os.path.split(path1)[0]
+path1 = os.path.join(path1,"","Task_1\movies.csv")
```

Comment:
use os.path.join() for joining Task_1\movies.csv

Reply:
I used that(os.path.jin()) to join it..